parser/src/main/scala/hmda/parser/fi/lar/LarCsvParser.scala

```diff
@@ -2,53 +2,57 @@ package hmda.parser.fi.lar
 
 import hmda.model.fi.lar._
 
-import scala.util.Try
+import scala.collection.immutable.ListMap
+import scalaz._
+import scalaz.Scalaz._
+import scala.util.{ Failure, Success, Try }
 
 object LarCsvParser {
   def apply(s: String): Either[List[String], LoanApplicationRegister] = {
-    val larErrors = checkLar(s)
-
-    if (larErrors.isEmpty) {
-      val values = s.split('|').map(_.trim)
-      val id = values(0).toInt
+    val values = s.split('|').map(_.trim)
+    val larErrors = checkLar(values.toList)
+    val validation = larErrors.disjunction
+    if (validation.isRight) {
+      val rSuccess = validation.toEither.right.get
```

Comment:
Maybe rename this variable to something like `convertedValues`, and put a newline after this one